cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -236,12 +236,32 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         },
 
         getIntroductionMessage: function() {
-            return StringUtils.interpolate(
+            var preview_text = '';
+            if (this.model.get('highlights_preview_only')) {
+                preview_text = StringUtils.interpolate(
+                    gettext(
+                        'We are testing this feature right now. So messages will not actually be sent at this time.'
```

Comment:
This feature is currently in testing. Course teams can enter highlights, but learners will not receive email messages.